src/plugins/discover/public/components/discover_grid/discover_grid.tsx

```diff
@@ -599,7 +604,7 @@ export const DiscoverGrid = ({
           className={classnames(
             className,
             'dscDiscoverGrid__table',
-            isPlainRecord ? 'dscDiscoverGrid__textLanguageMode' : 'dscDiscoverGrid__documentsMode'
+            'dscDiscoverGrid__documentsMode'
```

Comment:
Nit: we could probably remove `dscDiscoverGrid__documentsMode` now and move its styles to `dscDiscoverGrid__table` instead, but we could do this cleanup later in a different PR if you don't want to add to this one.

Reply:
Done https://github.com/elastic/kibana/pull/158623/commits/9e12704f7563f70113e2ffe2a8794d64582aa874